packages/sdk/src/modules/moduleFromEndpoints/types.ts

```diff
@@ -64,6 +64,11 @@ export type HTTPClient = (
   config: HTTPClientConfig
 ) => Promise<any>;
 
+/**
+ * Error handler abstraction.
+ */
+export type ErrorHandler = (error: any) => Promise<any>;
```

Comment:
I don't think that `error` would be enough to provide a solid error handling. User should be aware what was the method which caused the error, what params were passed etc

Reply:
can you list what is the most important for you to have in the error handling? I don't see any objections just want to know what to add.

Reply:
I think we could place here:
- error
- method name
- passed arguments
- httpClient

Reply:
Added.